src/rpc/common/Validators.cpp

```diff
@@ -196,4 +196,32 @@ CustomValidator SubscribeAccountsValidator =
         return MaybeError{};
     }};
 
+CustomValidator AssetValidator =
```

Comment:
Is this potentially something that can be unified with `ammAssetValidator` in LedgerEntry.h?

Reply:
Seems like `ammAssetValidator` in ledgerEntry.h is a better implementation because it uses rippled function `ripple::issueFromJson`. If rippled code changes, there'll be less chances of clio breaking. I'm going to put this method in the `validators.cpp` file and use it instead.